content/dojos/68-robots.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "#68 robots"
```

Comment:
Willen we hoofdletters beginnen of met kleine letters (zie eerste 2 en laatste 2 dojos):
![image](https://user-images.githubusercontent.com/7276252/91127660-ffdf3800-e6a6-11ea-8df3-e71dbdcfa626.png)

Reply:
Gekozen voor hoofdletters.